workflow/cron/operator.go

```diff
@@ -117,58 +116,68 @@ func getWorkflowObjectReference(wf *v1alpha1.Workflow, runWf *v1alpha1.Workflow)
 }
 
 func (woc *cronWfOperationCtx) persistUpdate() {
-	_, err := woc.cronWfIf.Update(woc.cronWf)
+	if woc.origCronWf.ResourceVersion != woc.cronWf.ResourceVersion {
```

Comment:
this is a guard-rail

Reply:
This might also not be necessary depending on https://github.com/argoproj/argo/pull/3883/files#r480375881